public/app/features/trails/BreakdownScene.tsx

```diff
@@ -141,7 +141,11 @@ export class BreakdownScene extends SceneObjectBase<BreakdownSceneState> {
         {loading && <div>Loading...</div>}
         <div className={styles.controls}>
           <Field label="By label">
-            <RadioButtonGroup options={labels} value={value} onChange={model.onChange} />
+            <Select
+              onChange={(val) => model.onChange(val.value || ALL_VARIABLE_VALUE)}
+              options={labels}
+              value={value}
+            />
```

Comment:
We should only switch to Select if there are many labels I think, I find it very useful to see all the label names.